### ISSUE TITLE
Windscribe service using ISP IP

### ISSUE THREAD

**Issue by grumpygeek** [JustOff/moonscribe#2]
Something seems to have changed recently.  Now it says "Connected to Proxy" but is using the local ISP IP address.  I submitted a ticket to Windscribe but they have not replied.  I can only use the browser extension because the Windows VPN program would never connect.  I use Moonscribe in Basilisk but I will test in PM if you think that would matter.  I can't test with any other ISP for the time being.

**Comment by JustOff** [comment]
Just checked, Moonscribe works well for me, both in Basilisk and Pale Moon. Perhaps you used the whole traffic limit?

PS: Please note, that Moonscribe is unofficial client, so they are unlikely to help.

**Comment by grumpygeek** [comment]
I still have full 10GB left.   Unfortunately, this is something I don't know how to troubleshoot well.  I noticed when I hover "Cruise Control" it shows a notice: "Location will be changed automatically when accessing a blocked resource".  But I havven't whitelisted Github and it isn't changing for this site, so that is not correct.  And I generally would not want to have to try access to blocked sites to find out if the proxy was going to work.

If you can't help, it's OK.  I didn't have antwhere else to post.

**Comment by JustOff** [comment]
>  I noticed when I hover "Cruise Control" it shows a notice

This notice is always shown there

> Now it says "Connected to Proxy" but is using the local ISP IP address.

How did you check that it really is?

**Comment by grumpygeek** [comment]
Oh, forgot to say, I didn't tell Windscribe I was using Moonscribe because I do have current Windscribe in Firefox; I just don't use that much.

I tried manually selecting server location (US East, Central, West, Canada East) and with all it still stays on local ISP.  So something is definitely different, but it could be at their end.

**Comment by JustOff** [comment]
>  I do have current Windscribe in Firefox

Do you have the same issue with it?

**Comment by grumpygeek** [comment]
I know it's closed now, just wanted to add ... Yes, sort of.  The current Web Extension in Firefox (and LibreWolf) does show local ISP IP sometimes when it also shows as connected to proxy.  But when I've checked recently, using BrowserLeaks.com, the proxy IP was the one reported back, not the local IP.  I have to stick to the few free proxy choices but I don't know if that matters.  Anyway, I gave up on using Moonscribe for now and use Librewolf or Firefox with the webext.  Both  of those are heavy users of RAM so I'd go back to Basilisk or PM if I got it working right again.
